assets/js/buttons-examples.js

```diff
@@ -415,6 +415,23 @@ $(document).ready(function () {
             },
           ],
         },
+        {
+          type: "toggle",
+          label: "Toggle button (on and off)",
+          classes: "toggle-btn",
+          shouldHaveRoundedType: false,
+          shouldHaveBlockType: false,
+          variations: [
+            {
+              classes: "toggle-on-btn",
+              text: "",
+            },
+            {
+              classes: "toggle-off-btn",
+              text: "",
+            },
+          ],
+        }
```

Comment:
There's a syntax error here please add a comma

Reply:
@shahednasser  I m in the middle of a really bad merge conflict situation. Should I just create a new pull request?